gateway/setup/generate_conf.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+# init
+
+echo
+echo
+echo "===============  GENERATE GATEWAY CONFIGURATION FILES ==============="
+echo
+echo
+
+
+HOST_CONF_PATH="$1"
+
+mkdir -p $HOST_CONF_PATH
+
+# generate ethereum file
+cp "$( cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )/../src/templates/ethereum.yml" "$HOST_CONF_PATH/ethereum.yml"
```

Comment:
This is difficult to maintain. You should use a `for $(find "$HOST_CONF_PATH" -name "*.yml")` loop to do a recursive copy.

Reply:
Understood.